chart/templates/aws-calico.yaml

```diff
@@ -15,7 +15,7 @@ spec:
     helm:
       values: |
         aws-calico:
```

Comment:
This needs to change to `calico` as well. Also [this](https://github.com/aws-samples/eks-blueprints-add-ons/blob/main/add-ons/aws-calico/values.yaml) file can be deleted.